libats-messaging/src/main/scala/com/advancedtelematic/libats/messaging/MessageListener.scala

```diff
@@ -28,24 +30,24 @@ object MessageListener {
       message.committableOffset.commitScaladsl().map(_ => msg())
   }
 
-  def buildSource[T](fromSource: Source[CommittableMsg[T], NotUsed],
-                     op: MsgOperation[T])
+  def buildSource[T](fromSource: Source[CommittableMsg[T], NotUsed], op: MsgOperation[T])
                     (implicit system: ActorSystem, ml: MessageLike[T]): Source[T, NotUsed] = {
     implicit val ec = system.dispatcher
 
-    fromSource
-      .mapAsync(3) { committableMsg =>
-        val msg = committableMsg.msg()
-        op(msg).map(_ => committableMsg)
-      }
-      .mapAsync(1) { committableMsg =>
-        committableMsg.commit()
-      }
+    fromSource.mapAsync(3) { committableMsg =>
```

Comment:
too many knobs is also bad, we are running this on small mesos agents anyway it's not like we have many available cores.